### ISSUE TITLE
Slade3 freeze when I close it (and use 100% of its core)

### ISSUE THREAD

**Issue by Bubuche87** [sirjuddington/SLADE#1045]
**SLADE Version**: (3.1.4) 
**Operating System**: (Windows)

**Issue Details**:
I downloaded the latest version of slade3 some days ago. 
When I try to close it, most of time (not every time, but way more than 50% of times) it doesn't stop, and starts to eat 25% CPU on a 4 core (so : goes into infinite loop somewhere).

Please note that **I don't have an internet connection when I use it**. It may be related to this (and invisible for most people as my situation is more and more rare).
Please note also that the previous version I was using didn't have this bug.
However, sadly, I can't tell which version it was. I remember I downloaded it first during 2016, and I don't think I updated it since then.
In that version the "home page" (the  thing you see when you open it) was a bit more colored. 
If you really need more details, I can try to browse every version since then until I find the one it was.

Note that this bug occurs even If I just open slade to check a texture (so I don't open the map editor, I don't open complex wad or multiple archives etc.)

**Comment by Bubuche87** [comment]
Ok, if there is no fix, is it possible to download older versions ? Not "2" or "1" version, older "3.x.x" versions ?

**Comment by Gaerzi** [comment]
Yes, here: [http://slade.mancubus.net/files/](http://slade.mancubus.net/files/).

**Comment by Bubuche87** [comment]
Thanks a lot. I'll download a previous version and test it tomorrow. 
Note that even if it solves the problem for me, it, of course, doesn't mean that the problem is solved in Slade.
